src/lib/RpcApi.ts

```diff
@@ -182,15 +182,14 @@ export default class RpcApi {
                 this._recoverState(state);
 
                 // Set result
-                result.kind = command;
```

Comment:
I did this because it was a type error in TS, but I'm not actually sure if this doesn't break something somewhere. I looked, but couldn't find code that uses it.

Reply:
Should be safe to remove it for now.